From 8f118816d7cf9b6279e8ff4113c4503ec69564c5 Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Sun, 5 Dec 2021 15:50:30 +0200
Subject: [PATCH 2/2] Add long int to toString

---
 include/mbgl/util/string.hpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/mbgl/util/string.hpp b/include/mbgl/util/string.hpp
index 9ed5a49e5..fbd6121d3 100644
--- a/include/mbgl/util/string.hpp
+++ b/include/mbgl/util/string.hpp
@@ -58,6 +58,11 @@ inline std::string toString(unsigned long long t) {
     return toString(static_cast<uint64_t>(t));
 }
 
+template <typename = std::enable_if<!std::is_same<int64_t, long int>::value>>
+inline std::string toString(long int t) {
+    return toString(static_cast<int64_t>(t));
+}
+
 template <typename = std::enable_if<!std::is_same<int64_t, long long>::value>>
 inline std::string toString(long long t) {
     return toString(static_cast<int64_t>(t));
-- 
2.32.0

